Tests/LogicTests/SantaCacheTest.mm

```diff
@@ -120,57 +110,105 @@ - (void)testThreading {
 }
 
 - (void)testCount {
-  auto sut = new SantaCache<uint64_t>();
+  auto sut = SantaCache<uint64_t, int>();
```

Comment:
I assume you wanted to test this with int?

Reply:
I did